content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -255,6 +256,169 @@ When the workloads are deployed, above, they send CSR Requests to Istiod which f
     $ kubectl delete ns bookinfo
     {{< /text >}}
 
+## Part 3: Using Custom CA with different signers for different workloads
+
+For this example, we use [open-source cert-manager](https://cert-manager.io).
+cert-manager has added [experimental Support for Kubernetes CertificateSigningRequests](https://cert-manager.io/docs/usage/kube-csr/) from v1.4.
+
```

Comment:
nit: for clarification, could we add "Each cert-signer is effectively a different CA. It is expected that workloads whose certificates are issued from the same cert-signer can talk MTLS to each other while workloads signed by different signers cannot."